src/cci/cci_util.c

```diff
@@ -772,6 +775,82 @@ ut_double_to_str (double value, char *str, int size)
   snprintf (str, size, "%.16f", value);
 }
 
+void
+ut_double_to_trailingzero_str (double value, char *str, int size)
+{
+  char double_str[MAX_DOUBLE_STRING];
+  char return_str[MAX_DOUBLE_STRING] = { '-', '0', '.', '\0', };
+  int exp_num, offset, p;
+
+  char *dot, *exp, *sp = return_str + 1;
+
+  sprintf (double_str, "%.16g", fabs (value));
+  dot = strchr (double_str, '.');
+  exp = strchr (double_str, 'e');
+
+  if (!exp)
+    {
+      sprintf (str, "%s%s", (value < 0) ? "-" : "", double_str);
```

Comment:
suggest

snprintf (str, size, "%s%s", (value < 0) ? "-" : "", double_str);

Reply:
thanks for your comment